tests/functional/artifacts/test_previous_version_state.py

```diff
@@ -443,7 +443,7 @@ def test_compare_results_current(self, project):
             current_run_results_schema_version == self.CURRENT_EXPECTED_RUN_RESULTS_VERSION
         ), "Sounds like you've bumped the run_results version and need to update this test!"
         # If we need a newly generated run_results, uncomment the following line and commit the result
-        self.generate_latest_run_results(project, current_run_results_schema_version)
+        # self.generate_latest_run_results(project, current_run_results_schema_version)
```

Comment:
Did I accidentally leave it un-commented from my PR?... The answer is yes, yes I did 😬 🤦 Thank you for catching this.